.github/workflows/release.yml

```diff
@@ -38,3 +38,35 @@ jobs:
           tag: ${{github.event.inputs.version}}
           token: ${{ secrets.GITHUB_TOKEN }}
           prerelease: ${{github.event.inputs.pre-release}}
+  notify-complete-success:
+    needs: [ release ]
+    name: Notify Release Completed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEB_HOOK }}
+          SLACK_COLOR: "#36a64f"
+          SLACK_USERNAME: CLI Release Bot
+          SLACK_ICON_EMOJI: ":ship:"
+          SLACK_TITLE: 'Homebrew Release Completed'
+          MSG_MINIMAL: actions url
+  notify-complete-fail:
+    if: ${{ failure() || cancelled() }}
+    needs: [ release ]
+    name: Notify Release Failed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.ALERT_SLACK_WEB_HOOK }}
+          SLACK_COLOR: "#ff3333"
+          SLACK_USERNAME: CLI Release Bot
+          SLACK_ICON_EMOJI: ":ship:"
+          SLACK_TITLE: "Twilio Cli"
```

Comment:
[nit]: `SLACK_TITLE: "Homebrew Release Failed" ` ? To be in sync with a successful case in line #54.